src/main/java/spiffe/api/svid/SpiffeEndpointChannelBuilder.java

```diff
@@ -107,10 +108,13 @@ private static void configureNativeSocketChannel(NettyChannelBuilder channelBuil
         checkNotNull(channelBuilder, "Channel builder is Null");
         if (SystemUtils.IS_OS_LINUX) {
             channelBuilder.eventLoopGroup(new EpollEventLoopGroup())
-                          .channelType(EpollDomainSocketChannel.class);
+                    // avoid Unknown channel option 'SO_KEEPALIVE'
+                    .withOption(ChannelOption.SO_KEEPALIVE, null)
+                    .channelType(EpollDomainSocketChannel.class);
```

Comment:
I patched the diff to my package (it might be a bit older version of the repo though), and i am getting this compile error in IDE:
```
Required type:
io.grpc.netty.shaded.io.netty.channel.ChannelOption
<T>
Provided:
io.netty.channel.ChannelOption
<Boolean>
reason: no instance(s) of type variable(s) T exist so that ChannelOption<Boolean> conforms to ChannelOption<T>
```
@maxlambrecht  Any idea why?

Reply:
what do you get if you run `./gradlew clean build`  ?

Reply:
yeah, this worked! lgtm